pkg/apis/openebs.io/upgrade/v1alpha1/upgraderesult.go

```diff
@@ -45,8 +46,19 @@ type UpgradeResult struct {
 
 // UpgradeResultConfig represents the entire config
 // of UpgradeResult i.e. same as task-executor job config
+// with single unit of upgrade in resource details. pv or sp
+// will be single unit of upgrade.
 type UpgradeResultConfig struct {
-	// TODO: Add Task-executor job config here
+	ResourceDetails
```

Comment:
Can we avoid this structure completely and make use of ExecutorConfig i.e. UpgradeConfig